ztest/ztest.go

```diff
@@ -199,7 +199,8 @@ func Run(t *testing.T, dirname string) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	for _, b := range bundles {
+	for _, bundle := range bundles {
+		b := bundle
```

Comment:
Nit: The usual idiom for this is to redeclare the same variable instead of introducing a new one.
```go
	for _, b := range bundles {
		b := b
```

Reply:
Ah, good to know. I'm not good at remembering these idioms.